packages/contracts-bedrock/src/L1/OPContractsManager.sol

```diff
@@ -363,6 +408,101 @@ contract OPContractsManager is ISemver {
         return output;
     }
 
+    function addGameType(AddGameInput[] memory _gameConfigs) external returns (AddGameOutput[] memory) {
+        if (address(this) == address(thisOPCM)) revert OnlyDelegatecall();
+
+        AddGameOutput[] memory outputs = new AddGameOutput[](_gameConfigs.length);
+        Blueprints memory bps = thisOPCM.blueprints();
+
+        for (uint256 i = 0; i < _gameConfigs.length; i++) {
```

Comment:
We could add some validation to ensure we don't see the same `disputeGameType` more than once.

Reply:
i'm near the maximum number of stack variables in this method and would prefer to leave that validation out unless it could cause a critical bug.

Reply:
You can always do something like push the game deployment to a helper method.  Just seems fairly easy to, for example, accidentally copy the wrong systemConfig value to one of the `_gameConfigs`, and then you think the update succeeded but it actually silently failed for one of your intended chains.

Reply:
fixed - it requires that you put the values in order. you can't have in-memory mappings in solidity to this was the cleanest way I could find to enforce this requirement.